src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -120,6 +120,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Get-PSHostProcessInfo tests.Verifies named pipe filepath is removed on process exit
```powershell
Expected path '\\.\pipe\PSHost.132322639863400373.2760.DefaultAppDomain.pwsh' to not exist, but it did exist.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Get-PSHostProcessInfo.Tests.ps1: line 99
99:         $psNamedPipePath | Should -Not -Exist
```